generators/client/files.js

```diff
@@ -0,0 +1,151 @@
+const jhipsterConstants = require('generator-jhipster/generators/generator-constants');
```

Comment:
Why are we overwriting the client generator. We should use the Client generator as such from the JHipster or else MHipster will become difficult to maintain.

Reply:
Unfortunately need to overwrite some of the files to be compatible w/ the standard output from micronaut, as we are not a 100% match on top of what Spring Boot outputs (nor should we really be; for example, it wouldn't make sense to match the nested structure of Spring Boot's `config` endpoint when our config is treated as a flat map). 

Our `health`, `metrics`, and auth endpoints also have slightly different output, thus I had to make some small tweaks to the frontend for all of those things.

I had a similar feeling to you at first but after speaking with @jameskleeh about it we both agreed this was the right course of action. I agree that it adds some maintenance weight, but hopefully by only overriding a few of the admin modules, I've minimized the impact as best as it can be minimized. I'm open to more suggestions on how to improve it though!

Reply:
This makes sense.